typescript-react.md

```diff
@@ -0,0 +1,26 @@
+# Typescript/React
+
+_Spec v1 (2021-11-05)_
+
+## Options
+
+There's two alternatives from which you can choose from:
+
+1. Create a generic one-page scroller where the hero element has a background video served from IPFS.
+2. Using the [Vanilla SDK](https://www.npmjs.com/package/@vanilladefi/sdk), create a Next.js page with the list of users utilizing React 18 and server components.
+
+## Requirements
+
+1. Use Typescript
+2. Document setting up the development environment with a README
+3. The website should be responsive
```

Comment:
When talking about requirements, let's be imperative: "Make website responsive."

Reply:
Good point.